packages/expo-dev-launcher/ios/EXDevLauncherController.h

```diff
@@ -27,6 +28,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol EXDevLauncherControllerDelegate <NSObject>
 
+@property (nonatomic, weak) RCTAppDelegate *rctAppDelegate;
```

Comment:
that seems not being used?
looks like we only use `[self.delegate rctAppDelegate]` which stands for `ExpoDevLauncherReactDelegateHandler`

Reply:
actually, `self.delegate` was an `EXDevLauncherControllerDelegate`, so that's why it was declared like this but I've just updated it to be `ExpoDevLauncherReactDelegateHandler `